item.rb

```diff
@@ -1,19 +1,20 @@
-require_relative 'date_handler'
-
+require_relative './date_handler'
 class Item
-  attr_accessor :publish_date, :genre
-  attr_reader :id, :source, :author, :label, :archived
+  attr_accessor :publish_date
+  attr_reader :id, :genre, :source, :label, :archived, :author
 
-  def initialize(publish_date, id = nil, archived: false)
-    @id = id || rand(1...1000)
+  def initialize(publish_date:, id: nil, archived: false)
+    @id = id || Random.rand(1...1000)
     @publish_date = DateHandler.from_string(publish_date)
     @archived = archived
-
     @source = []
-    @author = []
     @label = []
   end
 
+  def add_genre(genre)
+    @genre << genre
+  end
```

Comment:
While merging you accidentally added this back maybe, this method should be removed.
Also, don't forget to remove the add_author method also, since they are not arrays anymore.